src/panels/lovelace/entity-rows/hui-select-entity-row.ts

```diff
@@ -87,7 +87,7 @@ class HuiSelectEntityRow extends LitElement implements LovelaceRow {
       ></state-badge>
       <ha-paper-dropdown-menu
         .label=${this._config.name || computeStateName(stateObj)}
-        .disabled=${UNAVAILABLE_STATES.includes(stateObj.state)}
+        .disabled=${stateObj.state === UNAVAILABLE}
```

Comment:
Updating this check is 👍 

But what option is selected when it is unknown? It should be 'no option'. Does `ha-paper-dropdown-menu` take care of that automatically?

Reply:
Unavailable:

![image](https://user-images.githubusercontent.com/195327/123331181-3c85f500-d53f-11eb-8f3b-349febbcf389.png)

Unknown:

![image](https://user-images.githubusercontent.com/195327/123331234-4f98c500-d53f-11eb-8f89-8bfe052fe807.png)